src/main/resources/assets/arcanuscontinuum/lang/fr_fr.json

```
@@ -0,0 +1,334 @@
+
+	"spell_book.arcanuscontinuum.weight": "Poids",
+	"spell_book.arcanuscontinuum.weight.none": "Auncun",
+	"spell_book.arcanuscontinuum.weight.very_light": "Très Léger",
+	"spell_book.arcanuscontinuum.weight.light": "Léger",
+	"spell_book.arcanuscontinuum.weight.medium": "Moyen",
+	"spell_book.arcanuscontinuum.weight.heavy": "Lourd",
+	"spell_book.arcanuscontinuum.weight.very_heavy": "Très Lourd",
+	"spell_book.arcanuscontinuum.mana_cost": "Coût de Mana",
+	"spell_book.arcanuscontinuum.mana_multiplier": "Mana Multiplier",
+	"spell_book.arcanuscontinuum.potency_modifier": "Potency Modifier",
+	"spell_book.arcanuscontinuum.cool_down": "Refroidissement",
+	"spell_book.arcanuscontinuum.seconds": "s",
+
+	"staff.arcanuscontinuum.invalid_data": "!!! DONNEES INVALIDES !!!",
+	"staff.arcanuscontinuum.primary_color": "Couleur des cristeaux",
+	"staff.arcanuscontinuum.secondary_color": "Couleur du bâton",
+
+	"spell.arcanuscontinuum.too_many_components": "Trop de composants!",
+	"spell.arcanuscontinuum.too_low_level": "Niveau de sorcellerie trop bas!",
+	"spell.arcanuscontinuum.not_enough_mana": "Pas assez de Mana!",
+
+	"door.arcanuscontinuum.access_granted": "Accès accordé...",
+	"door.arcanuscontinuum.not_owner": "Cette porte ne vous appartiens pas",
+	"door.arcanuscontinuum.say_magic_word": "Cette porte a besoin d'un mot de passe",
+	"door.arcanuscontinuum.password_set": "Le mot de passe defini est %s",
+
+	"screen.arcanuscontinuum.tooltip.undo": "Annuler",
+	"screen.arcanuscontinuum.tooltip.redo": "Refaire",
+	"screen.arcanuscontinuum.tooltip.component_count": "Spell Components",
+	"screen.arcanuscontinuum.tooltip.change_screens": "Changer de Mode",
+	"screen.arcanuscontinuum.tooltip.cycle_up": "Skin précedent",
+	"screen.arcanuscontinuum.tooltip.cycle_down": "Skin suivant",
+
+	"wizard_dialogue.arcanuscontinuum.no_wizard_armour": "Reviens me voir quand tu ressemblera plus à un sorcier...",
+	"wizard_dialogue.arcanuscontinuum.put_on_robes": "Peut être que porté une robe de sorcier vous aidera...",
+	"scroll_of_knowledge.arcanuscontinuum.level_up": "Vous vous sentez un peu plus intelligent...",
+
+	"death.attack.arcanuscontinuum.magic": "%s a été tué par %s par la magie",
+	"death.attack.arcanuscontinuum.magic.item": "%s a été tué par %s utilisant la magie avec %s",
+
+	"attribute.name.generic.arcanuscontinuum.max_mana": "Mana Maximum",
+	"attribute.name.generic.arcanuscontinuum.mana_regen": "Regeneration de mana",
+	"attribute.name.generic.arcanuscontinuum.burnout_regen": "Epuisement de Mana",
+	"attribute.name.generic.arcanuscontinuum.mana_lock": "Réstriction de Mana",
+	"attribute.name.generic.arcanuscontinuum.spell_potency": "Puissance des Sorts",
+  "attribute.name.generic.arcanuscontinuum.mana_cost": "Coût de Mana",
+	"attribute.name.generic.arcanuscontinuum.magic_resistance": "Résistance Magique Rés",
+  "attribute.name.generic.arcanuscontinuum.spell_cool_down": "Refroidissement du sort",
```

```suggestion
  "attribute.name.generic.arcanuscontinuum.spell_cool_down": "Délai entre les sorts",
```
Based on https://fr.minecraft.wiki/w/D%C3%A9g%C3%A2ts#D%C3%A9lai_entre_les_coups